pom.xml

```diff
@@ -280,6 +308,64 @@
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.spark</groupId>
+        <artifactId>spark-core_2.11</artifactId>
+        <version>${spark2.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.esotericsoftware.reflectasm</groupId>
```

Comment:
what is this? do we need it?

Reply:
The exclusions list is the same in most plugins which run streaming integration tests.
https://github.com/search?q=org%3Adata-integrations+com.esotericsoftware.reflectasm&type=Code


If we don't do exclusions integration tests runs will fail due to conflicting libs. Probably com.esotericsoftware.reflectasm depends on some libs which conflict with CDAP classes.